llvm/lib/CodeGen/GlobalISel/CombinerHelper.cpp

```diff
@@ -4100,6 +4100,14 @@ void CombinerHelper::applyBuildFn(
   MI.eraseFromParent();
 }
 
+void CombinerHelper::applyBuildFnMO(const MachineOperand &MO,
+                                    BuildFnTy &MatchInfo) {
+  MachineInstr *Root = getDefIgnoringCopies(MO.getReg(), MRI);
+  Builder.setInstrAndDebugLoc(*Root);
```

Comment:
Setting the insert point should now be unnecessary in apply functions

Reply:
Rebasing made it end in https://github.com/llvm/llvm-project/pull/87115